dnf/cli/cli.py

```diff
@@ -251,6 +251,7 @@ def gpgsigcheck(self, pkgs):
         :return: non-zero if execution should stop due to an error
```

Comment:
I think we should remove "return". The text is not true. It always returns 0 or exception is raised.
Return value is not used anywhever.